### PR TITLE
feat: card component added

### PR DESCRIPTION
**Pull request by Rishabh-078**
Different types of card components are added, some of them being: basic card, horizontal card, vertical card, the card with dismiss button, and the card with text overlay
a
.

### CODE REVIEW

File: documentation.html [hunk 1]
```diff
@@ -272,6 +275,143 @@ <h5>Examples of Link, Float Button and Icon Buttons: </h5>
...
+                        sandbox="allow-scripts allow-same-origin" class="vertical-iframe">
+                    </iframe>
+
+                    <!-- Vertical-Card with Dismiss-->
+                    <div class="card card-vertical-container card-vertical-dismiss">
+                        <div class="image-badge">
+                            <img src="https://www.kingsanddukes.com/images/armani-jeans-mens-purple-polo-shirt-p4264-20827_image.jpg"
+                                alt="" />
```
Comment (Naman-Saxena1) [inline documentation.html]: alt tags are empty at some places.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: documentation.html [hunk 2]
```diff
@@ -272,6 +275,143 @@ <h5>Examples of Link, Float Button and Icon Buttons: </h5>
...
+                        <img class="basic-img" src="https://material-components.github.io/material-components-web-catalog/static/media/photos/3x2/2.jpg"
+                            alt="" />
+                        <div class="basic-card-content">
+                            <h3>Our Changing Planet</h3>
+                            <p>by Kurt Wagner</p>
+                            <p>Visit ten places on our planet that are undergoing the biggest changes today.</p>
+                        </div>
+                        <div class="basic-card-elements">
+                            <a class="card-link" src="">READ</a>
+                            <a class="card-link" src="">BOOKMARK</a>
```
Comment (Sahil25061999) [inline documentation.html]: a tag will contain href

File: lib/components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+        "card-vertical vertical-iframe"
+        "card-ver-dismiss ver-dismiss-iframe"
+        "card-ver-overlay ver-overlay-iframe";
+}
+
+.card {
+    border: 1px solid var(--secondary-color);
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    position: relative;
+    border-radius: 0.5rem;
```
Comment (Sahil25061999) [inline lib/components/Card/card.css]: if value is less than 10px you can use px, but its upto you.

**Review by Sahil25061999 — COMMENTED** [review]
Rest of the code looks good can be merged

File: documentation.html (hunk 1 above)
Comment (Rishabh-078) [reply documentation.html]: Thanks, had rectified it.

File: documentation.html (hunk 2 above)
Comment (Rishabh-078) [reply documentation.html]: Made required changes, thanks for pointing it out

File: lib/components/Card/card.css (hunk 3 above)
Comment (Rishabh-078) [reply lib/components/Card/card.css]: Have changed it to px.

**Pull request merged**